content/rest-api/beta.textile

```diff
@@ -311,3 +310,61 @@ response body
   ]
 }
 ```
+
+h4(#batch-presence-response). Batch responses with the request method
+
+When using the request method in a library, you'll need to handle the potential responses for success, partial success, and failure.
+
+For a response object @response@, @response.success@ will be true if the batch presence was successful, or @false@ if at least one presence request failed. @response.statusCode@ will be 200 for complete success, 400 for partial success, and 401 for an expected failure. @response.errorCode@ will contain the Ably error code, and @response.errorMessage@ will contain details of the error.
+
+For successful requests, @response.items@ will contain a list of the responses for each individual channel from which presence has been requested.
+
+For partially successful requests, @response.items.batchResponse@ will contain a list of the individual channel's results, be it an error or a success.
+
+Below would be an example of how to deal with these:
+
+```[javascript]
+var ablyRest = new Ably.Rest({ key: '{{API_KEY}}' })
+var content = { "channel": "channel1,channel2" }
+ablyRest.request('GET', '/presence', content, null, {}, function(err, response) {
+  if(err) {
+    // Throw error
+  } else {
+    if(response.success) {
+      // If complete success
+      for(i = 0; i < response.items.length; i++) {
+        // Each response item will be roughly of the style:
+        /*
+          {
+            "channel": "channel1",
+            "presence": [
+              { "action": 1, "clientId": "CLIENT1" },
+              { "action": 1, "clientId": "CLIENT2" }
+            ]
+          }
+        */
+      }
+    } else {
+      if(response.items[0].batchResponse) {
```

Comment:
The definitive way to check for a partial success is to check for the `40020` batch error code. You should do this before looking in `batchResponse`